examples/confirm_all_snippets.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/local/bin/python3
+# encoding: utf-8
+'''
+Created on October 3, 2019
+
+@author: ylei
+
+Confirm all snippet matched automatically.
```

Comment:
I've said it before, and will say it again, confirming or marking snippet matches as reviewed in an automated fashion is a very risky thing to do. Snippet matches should be reviewed by a person who has sufficient knowledge of the project to understand when someone has copied intellectual property that puts the project at (legal) risk. This program blindly marks snippet matches as reviewed when they, in fact, have not been reviewed (by a person). Please remove this program from the pull request. I can approve the other parts but not this one.

Reply:
I will remove the auto confirming snippet stuff.